eng/pipelines/common/platform-matrix.yml

```diff
@@ -489,6 +489,7 @@ jobs:
       helixQueuesTemplate: ${{ parameters.helixQueuesTemplate }}
       variables: ${{ parameters.variables }}
       osGroup: wasi
+      osSubGroup: ${{ parameters.osSubGroup }}
```

Comment:
Needed (also below)?

Reply:
Hmm, I meant that it is not clear it is needed... Since we already have a name suffix (the current name for the jobs has `multithread` duplicated: `Build wasimultithread-wasm windows debug Multithreaded`).